tests/test_general.py

```diff
@@ -620,3 +621,16 @@ def test_2635():
     page.clean_contents()  # clean page
     pix2 = page.get_pixmap()  # pixmap after cleaning
     assert pix1.samples == pix2.samples  # assert equality
+
+
+def test_resolve_names():
+    """Test PDF name resolution."""
+    # guard against wrong PyMuPDF architecture version
+    if not hasattr(fitz.Document, "resolve_names"):
+        print("PyMuPDF version does not support resolving PDF names")
+        return
```

Comment:
[This is not a problem with this PR.] Might be slightly better to explicitly test PyMuPDF version here, just in case `resolve_names()` is accidently deleted or renamed. We have a `mupdf_version_tuple`, i'll look into adding a `pymupdf_version_tuple` or similar.